uat/steps/gdk_cli.py

```diff
@@ -0,0 +1,69 @@
+from behave import step, register_type, use_step_matcher
+from parse_type import TypeBuilder
+
+
+# ----------------------------------------------------------------------------
+# CHANGE STEP MATCHER FROM DEFAULT parse MODE TO cfparse MODE
+# ----------------------------------------------------------------------------
+use_step_matcher("cfparse")
+
+
+# ------------------------------------------------------------------------
+# USER-DEFINED TYPES:
+# ------------------------------------------------------------------------
+component_command_type = TypeBuilder.make_choice(["list", "init", "build", "publish"])
+register_type(CommandType=component_command_type)
+
+output_type = TypeBuilder.make_choice(["quietly", "silently"])
+register_type(OutputType=output_type)
+
+register_type(String=str)
+
+
+# ----------------------------------------------------------------------------
+# STEPS:
+# ----------------------------------------------------------------------------
+@step("we run {command:CommandType} command")
```

Comment:
I think it is better to have a step that takes in full CLI command instead of passing them as command and args.
Eg. 
When `I run the command "component list --template"` 

Otherwise, it'd be hard to maintain if there are multiple levels of sub-commands. 
Eg. `gdk component create template --some-example`

Reply:
Changing it to `we run gdk component list --template`